frontend/src/v4/routes/viewerGui/components/sequences/sequences.component.tsx

```diff
@@ -60,21 +62,44 @@ interface IProps {
 	draggablePanels: string[];
 	toggleLegend: () => void;
 	resetLegendPanel: () => void;
+	openOnToday: boolean;
 }
 
 const da =  new Date();
 
 const SequenceDetails = ({
-	startDate, endDate, selectedDate, selectedEndingDate, setSelectedDate, stepInterval, stepScale, setStepInterval,
-	setStepScale, currentTasks, loadingFrameState, loadingViewpoint, rightPanels, toggleActivitiesPanel,
+	startDate, endDate, selectedDate, selectedStartDate, selectedEndingDate, setSelectedDate, stepInterval, stepScale, setStepInterval,
+	setStepScale, currentTasks, loadingFrameState, loadingViewpoint, rightPanels, toggleActivitiesPanel, openOnToday,
 	fetchActivityDetails, onPlayStarted, frames, isActivitiesPending, toggleLegend, draggablePanels
-}) => (
+}) => {
+	const [dateToUse, setDateToUse] = useState(null);
```

Comment:
When I enter the player with the toggle off, I get an empty component:

![empty component](https://github.com/3drepo/3drepo.io/assets/42032433/19e18b42-e4e2-4371-b555-2dc96a34e40d)